obspy/imaging/scripts/scan.py

```diff
@@ -73,23 +73,23 @@ def compressStartend(x, stop_iteration):
 
 
 def parse_file_to_dict(data_dict, samp_int_dict, file, counter, format=None,
-                       verbose=False, ignore_links=False):
+                       verbose=False, quiet=False, ignore_links=False):
     from matplotlib.dates import date2num
     if ignore_links and os.path.islink(file):
-        print("Ignoring symlink: %s" % (file))
+        if verbose or not quiet:
```

Comment:
Oh, I misread somehow. For some reason, I thought it originally had a `if verbose:` there.